nemo/collections/nlp/models/intent_slot_classification_refactor/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo.collections.nlp.models.intent_slot_classification_refactor.intent_slot_classification_model import (
```

Comment:
the reason for this is that there is a `nemo.collections.nlp.models.intent_slot_classification` module (which I refactored from) that cannot yet be deprecated, since it's used by others downstream